lib/matplotlib/tests/test_axes.py

```diff
@@ -9733,6 +9733,39 @@ def test_bar_shape_mismatch():
         plt.bar(x, height)
 
 
+def test_caps_color():
+
+    # Creates a simple plot with error bars and a specified ecolor
+    x = np.linspace(0, 10, 10)
+    y = np.sin(x)
+    yerr = 0.1
+    mpl.rcParams['lines.markeredgecolor'] = 'green'
+    ecolor = 'red'
+
+    fig, ax = plt.subplots()
+    errorbars = ax.errorbar(x, y, yerr=yerr, ecolor=ecolor, fmt='o', capsize=5)
```

Comment:
```suggestion
    mpl.rcParams['lines.markeredgecolor'] = 'green'
    ecolor = 'red'

    fig, ax = plt.subplots()
    errorbars = ax.errorbar(x, sin(x), yerr=0.1, ecolor=ecolor)
```
- Inlining the only-once used parameters make the code more compact.
- The `fmt` and `capsize` parameters don't have an influence. Leaving them out keeps the code more readable.